app/src/main/java/org/sil/storyproducer/activities/BaseActivity.kt

```diff
@@ -18,7 +18,6 @@ import org.sil.storyproducer.controller.MainActivity
 import org.sil.storyproducer.controller.RegistrationActivity
 import org.sil.storyproducer.controller.SelectTemplatesFolderController
 import org.sil.storyproducer.controller.SelectTemplatesFolderController.Companion.SELECT_TEMPLATES_FOLDER
-import org.sil.storyproducer.controller.SelectTemplatesFolderController.Companion.SELECT_TEMPLATES_FOLDER_AND_ADD_DEMO
```

Comment:
This is fine for now. But FYI in the future I may ask for an additional Welcome dialog variant that does add the demo to the folder.

Reply:
I'll add this back so we can add back quickly if needed!

Reply:
I left the declaration of this in SelectTemplatesFolderController so we're good here